scripts/install.sh

```diff
@@ -103,3 +111,7 @@ jupyter labextension install packages/nteract-data-explorer
 
 # 7. List installed extensions
 jupyter labextension list
+
+echo -e "If you haven't set up your own conda and node,\nexecute the following commands to use ${GREEN}conda${PLAIN} and ${GREEN}node${PLAIN} installed by this script:"
```

Comment:
I guess we can check whether the user has installed conda/node?

Reply:
It does have the process in the beginning of the script.